Content.Client/NewFrontier/Latejoin/VesselListControl.xaml.cs

```diff
@@ -44,6 +44,10 @@ protected override void Dispose(bool disposing)
 
     private int DefaultComparison(NetEntity x, NetEntity y)
     {
+        if (_gameTicker.StationNames[x] == "Frontier Outpost" && _gameTicker.JobsAvailable[x].ContainsKey("Passenger"))
+        {
+            return 0;
```

Comment:
I don't see how that would work. The comparison function f(x, y) should return a positive value when x is greater than y, negative when x is lesser than y, and 0 when they're equal. In other words, it's the difference between x and y. In your case the function says "x and y are equal" if and only if x is the frontier outpost.